src/components/card-interactive/CardRadio.tsx

```diff
@@ -0,0 +1,239 @@
+import * as React from 'react';
+import cx from 'classnames';
+import Radio from '../form-elements/radio/Radio';
+import generateRandomString from '../../js/generateRandomString';
+import {useCardRadioGroupContext} from './CardRadioGroupContext';
+import type {StyleType} from './types';
+
+export interface CardRadioPropsType
+  extends React.ComponentPropsWithoutRef<'div'> {
+  /**
+   * Required string. Value of the CardRadio input.
+   */
+  value: string;
+
+  /**
+   * Optional boolean. Whether the Radio is required.
+   * @default false
+   */
+  required?: boolean;
+
+  /**
+   * Optional boolean. Whether the Radio is disabled.
+   * @default false
+   */
+  disabled?: boolean;
+
+  /**
+   * Optional boolean. Whether the Radio is invalid.
+   * @default false
+   */
+  invalid?: boolean;
+
+  /**
+   * Optional string. ID of the Radio.
+   */
+  id?: string;
+
+  /**
+   * Optional string. Variant of the card. Default is 'outline'.
+   */
+  variant?: 'solid' | 'outline';
+
+  /**
+   * Optional string. Color of the card. Default is 'dark'.
+   */
+  color?: 'light' | 'dark';
+
+  /**
+   * Optional string. Additional class names.
+   */
+  className?: string;
+
+  /**
+   * Optional React.ReactNode. Children of the card. This is the place where label should be used and connected to the card.
+   * @example <CardRadio>Card content</CardRadio>
+   */
+  children?: React.ReactNode;
+
+  /**
+   * Optional string. Width of the card.
+   * @default auto
+   * @example <CardRadio width="100px" />
+   */
+  width?: React.CSSProperties['width'];
+
+  /**
+   * Optional string. Height of the card.
+   * @default auto
```

Comment:
This directive describes JS prop, but the default value is set in only in CSS, so maybe we should remove it?